data_runner/README.md

```diff
@@ -0,0 +1,41 @@
+# Twenty Forty Eight
```

Comment:
Datadog's public documentation has a particular tone, and a lot of the information in this readme is background that might be interesting to a developer, but isn't pertinent to someone who just wants to use this. Since that information is available in the original repo, I propose that the readme here be more like the following:

```
# Data Runner

## Overview

Data Runner is a game in which your character searches a library environment for a player-specified metric. When the character finds the specified metric, the metric's value is added to the player's score. You can add Data Runner to your Datadog dashboard as a widget. 

For more details about Data Runner, see [stuartlangridge/data-runner][1] on GitHub.

## Setup

1. To display Data Runner on your dashboard, open the [dashboard][2] to which you want to add your widget.

2. Use the **+ Add Widgets** button to open a list of available widgets. Drag and drop the Data Runner widget into your desired position on the dashboard.

3. Choose a metric for the game character to find.

## Support

[Contact Datadog support][3] for help.

To build your own Datadog app, refer to the [Datadog apps developer documentation][4].

[1]: https://github.com/stuartlangridge/data-runner
[2]: https://app.datadoghq.com/dashboard/lists
[3]: https://www.datadoghq.com/support/
[4]: https://docs.datadoghq.com/developers/datadog_apps
```